ktor-server/ktor-server-core/api/ktor-server-core.api

```diff
@@ -2,29 +2,31 @@ public final class io/ktor/server/application/Application : io/ktor/server/appli
 	public fun <init> (Lio/ktor/server/application/ApplicationEnvironment;)V
 	public final fun dispose ()V
 	public fun getCoroutineContext ()Lkotlin/coroutines/CoroutineContext;
-	public fun getEnvironment ()Lio/ktor/server/application/ApplicationEnvironment;
 }
 
-public abstract interface class io/ktor/server/application/ApplicationCall {
-	public abstract fun getApplication ()Lio/ktor/server/application/Application;
-	public abstract fun getAttributes ()Lio/ktor/util/Attributes;
-	public abstract fun getParameters ()Lio/ktor/http/Parameters;
+public abstract interface class io/ktor/server/application/ApplicationCall : io/ktor/server/application/Call {
```

Comment:
It's still confusing to have io/ktor/server/application/ApplicationCall, io/ktor/server/application/Call
let's discuss